src/components/App/App.js

```diff
@@ -22,11 +22,14 @@ class App extends React.Component {
       }
     ]
   };
+
+  onClickDone = isDone => Console.log(isDone);
```

Comment:
Codacy found an issue: ['Console' is not defined.](https://app.codacy.com/gh/WebHeroSchool/tr-todo/pullRequest?prid=5883512)